feature-libs/cart/base/styles/components/_cart-item-list.scss

```diff
@@ -1,6 +1,18 @@
 %cx-cart-item-list {
```

Comment:
spacing is off in mobile view: 
<img width="809" alt="image" src="https://user-images.githubusercontent.com/34665674/224355586-3b29057e-aca5-475e-8e1d-172c2adf2bb1.png">